pkg/cvo/upgradeable.go

```diff
@@ -81,12 +82,14 @@ func (optr *Operator) setUpgradeableConditions() {
 	var conds []configv1.ClusterOperatorStatusCondition
 	var reasons []string
 	var msgs []string
+	klog.V(2).Infof("Checking upgradeability conditions")
```

Comment:
> I'm not sure whether I want to keep the Checking upgradeability conditions line. The "check was done" will be always represented by either one or more Upgradeability condition failed or a All upgradeability checks were successful, so one additional line is perhaps redundant (but there's a part of my brain that likes the 'something starts' and 'something finished' pairs).
> 
>  https://github.com/openshift/cluster-version-operator/pull/886#issuecomment-1387268750

I am not sure as well. I can think of some upsides and downsides. But I would maybe leave the logging here.

---

The `pkg/cvo/availableupdates.go` does print similar information (meaning "_I am updating this stuff_") but it provides additional information on why the update happened if it happened.

https://github.com/openshift/cluster-version-operator/blob/3f3204ceb4aa505a161db74214fc1fe3f40e426f/pkg/cvo/availableupdates.go#L49-L63

Reply:
I guess a good consensus is to log the `Checking upgradeability conditions` bit on higher verbosity

Reply:
That's also a possibility. For the time being I am fine with leaving the change in the commit as is.